java/client/src/main/java/glide/api/models/commands/FT/FTSearchOptions.java

```diff
@@ -82,31 +82,43 @@ void limit(Pair<Integer, Integer> limit) {}
 
         void count(boolean count) {}
 
-        void identifiers(Map<GlideString, GlideString> identifiers) {}
+        public FTSearchOptionsBuilder identifiers(Map<GlideString, GlideString> identifiers) {
+            this.identifiers$value = identifiers;
```

Comment:
This overwrites previously set identifiers and return fields. Is it ok? If not, use `putAll` or another method to copy data.